### DISCUSSION TITLE
How I can get errors on version 7?

### DISCUSSION THREAD

**Discussion by tassiorego** [react-hook-form/react-hook-form#4708] [General] [answered]
When I try get the errors in the new version I got the follow error

`Property 'errors' does not exist on type 'UseFormReturn<FormData>'. `

 ```js
   const { register, handleSubmit, errors } = useForm<FormData>();
```

Does not work with `errors`

**Comment by bluebill1049** [comment, marked as answer]
formState: { errors }

**Reply by bluebill1049** [reply]
have a look at the migration guide.

**Reply by tassiorego** [reply]
Ok. I will look that.

Thanks!

**Reply by djyde** [reply]
> have a look at the migration guide.

https://react-hook-form.com/api/useform/seterror isn't updated

**Reply by bluebill1049** [reply]
thanks @djyde

**Reply by AlbanaAhmeti** [reply]
i have the same problem, did you find any solutin?
